src/main/java/com/icthh/xm/ms/configuration/utils/ConfigPathUtils.java

```diff
@@ -1,26 +1,59 @@
 package com.icthh.xm.ms.configuration.utils;
 
+import static com.icthh.xm.commons.config.client.service.TenantConfigService.DEFAULT_TENANT_CONFIG_PATTERN;
 import static com.icthh.xm.commons.tenant.TenantContextUtils.getRequiredTenantKeyValue;
 import static com.icthh.xm.ms.configuration.config.Constants.CONFIG;
 import static com.icthh.xm.ms.configuration.config.Constants.TENANTS;
 
 import com.icthh.xm.commons.tenant.TenantContextHolder;
 import lombok.experimental.UtilityClass;
+import org.springframework.util.AntPathMatcher;
 
 import java.util.List;
+import java.util.Map;
+import java.util.Optional;
 import java.util.stream.Collectors;
 
 @UtilityClass
 public final class ConfigPathUtils {
 
     private static final int MAX_PATHS_TO_PRINT = 3;
+    private final AntPathMatcher matcher = new AntPathMatcher();
+    private static final String TENANT_NAME = "tenantName";
+    private static final String TENANT_PREFIX = CONFIG + TENANTS + "/";
```

Comment:
May be it is better to use Path without ending `/`. Then all folders starts from '/%name%'.

Reply:
it's internal constant, and all usage require / on end